spec/httparty/connection_adapter_spec.rb

```diff
@@ -191,6 +200,19 @@
           allow(Net::HTTP).to receive_messages(new: http)
           adapter.connection
         end
+
+        it "should not set the write_timeout" do
+          http = double(
+            "http",
+            :null_object => true,
```

Comment:
Please, use colon notation for symbol keyword arguments

Reply:
Hey @TheSmartnik thanks for the comments 👍 I addressed all of them, but this one. I would like to discuss one thing here. 

Due to the fact that the doubles contain names with special characters (e.g. `:use_ssl= => false`) converting them to Ruby 1.9 hash syntax would require using `'` to wrap them. Not sure if it is worth doing in this case. I am totally fine with leaving the current syntax.

Besides that, feel free to take a look at the pushed commits. The code looks better now IMO.